reproducibility_project/clean_by_engine.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+echo Removing jobs from all engines but $1
+for ENGINE in cassandra mcccs gomc gromacs hoomd lammps-VU lammps-UD
+do
+    if [[ $ENGINE != $1 ]]
+    then
+        signac rm $(signac find engine $ENGINE)
```

Comment:
Is there a verbose option for this? I think we would want to be verbose in this case so we see what is removed. Unless `signac rm` does this by default

Reply:
The verbose option for rm just echos the job id, which I don't find super helpful.

Reply:
Makes sense to me. I think this is pretty good to go then.